modules/ipp/scanner.go

```diff
@@ -0,0 +1,206 @@
+// Package ipp provides a zgrab2 module that scans for ipp.
+// TODO: Describe module, the flags, the probe, the output, etc.
+package ipp
+
+//TODO: Clean up these imports
+import (
+	//"bytes"
+	"encoding/binary"
+	//"errors"
+	//"fmt"
+	//"io"
+	"net/http"
+	"strconv"
+	"strings"
+	//"net"
+	//"net/url"
+	//"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zgrab2"
+)
+
+const (
+	CONTENT_TYPE string = "application/ipp"
+)
+
+//TODO: Tag relevant results and exlain in comments
+// ScanResults instances are returned by the module's Scan function.
+type ScanResults struct {
+	//TODO: Include a full response or at least a blob in the data
+	//Response *http.Response `json:"response,omitempty"`
```

Comment:
This would be nice, at least for verbose mode. Including the requests could be interesting, too.